src/main/java/uk/gov/pay/ledger/transaction/search/model/SettlementSummary.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.pay.ledger.transaction.search.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Objects;
+
+import static uk.gov.pay.commons.model.ApiResponseDateTimeFormatter.ISO_INSTANT_MILLISECOND_PRECISION;
+
+public class SettlementSummary {
+    private ZonedDateTime captureSubmitTime, capturedTime;
+
+    public SettlementSummary(ZonedDateTime settlementSubmittedTime, ZonedDateTime settledTime) {
+        captureSubmitTime = settlementSubmittedTime;
+        capturedTime = settledTime;
+    }
+
+    public static SettlementSummary ofValue(ZonedDateTime settlementSubmittedTime, ZonedDateTime settledTime) {
+        return new SettlementSummary(settlementSubmittedTime, settledTime);
```

Comment:
As this constructor is `public` could the code calling `ofValue` not directly instantiate the object?